tests/integration/example/tests/sample2/sample2_test.go

```diff
@@ -64,7 +64,8 @@ func TestEcho(t *testing.T) {
 	log.Printf("%s echo test succeeded!", testEM.GetID())
 }
 
-func TestMetrics(t *testing.T) {
+// Disabled for being flaky in prow.
+func NoTestMetrics(t *testing.T) {
```

Comment:
It might be a better practice to call `t.Skip("skipping to unblock prow.")` inside the function

Reply:
Thanks for the comment Charles, I'll just close this for #6752.